src/Sarif.UnitTests/Visitors/RebaseUriVisitorTests.cs

```diff
@@ -95,50 +93,21 @@ public void RebaseUriVisitor_VisitRun_UpdatesBaseUriDictionaryWhenPresent()
             Random random = RandomSarifLogGenerator.GenerateRandomAndLog(this.output);
 
             Run oldRun = RandomSarifLogGenerator.GenerateRandomRun(random);
-            oldRun.Properties = new Dictionary<string, SerializedPropertyInfo>();
             RebaseUriVisitor rebaseUriVisitor = new RebaseUriVisitor(srcRoot, srcRootUri);
 
             Dictionary<string, Uri> oldDictionary = new Dictionary<string, Uri>() { { bldRoot, bldRootUri } };
-
-            oldRun.Properties.Add(RebaseUriVisitor.BaseUriDictionaryName, RebaseUriVisitor.ReserializePropertyDictionary(oldDictionary));
+            oldRun.OriginalUriBaseIds = oldDictionary;
 
             Run newRun = rebaseUriVisitor.VisitRun(oldRun);
 
-            newRun.Properties.Should().ContainKey(RebaseUriVisitor.BaseUriDictionaryName);
-
-            Dictionary<string, Uri> baseUriDictionary = RebaseUriVisitor.DeserializePropertyDictionary(newRun.Properties[RebaseUriVisitor.BaseUriDictionaryName]);
+            IDictionary<string, Uri> baseUriDictionary = newRun.OriginalUriBaseIds;
 
             baseUriDictionary.Should().ContainKey(srcRoot);
             baseUriDictionary[srcRoot].Should().BeEquivalentTo(srcRootUri);
             baseUriDictionary.Should().ContainKey(bldRoot);
             baseUriDictionary[bldRoot].Should().BeEquivalentTo(bldRootUri);
```

Comment:
Nit--this pattern was just because we had to retrieve the dictionary to get it as a dictionary and that was overly verbose.  Can we just do the checks on the property since it's now part of the run?